mozilla_schema_generator/glean_ping.py

```diff
@@ -89,12 +90,9 @@ def get_probes(self) -> List[GleanProbe]:
         return [GleanProbe(_id, defn, pings=pings) for _id, defn in probes]
 
     def get_pings(self) -> Set[str]:
-        probes = [GleanProbe(_id, defn) for _id, defn in self._get_probe_defn_list()]
```

Comment:
Can you also remove the `_get_probe_defn_list` function, since `get_pings` was the only reason it was pulled out as a separate piece? It can be consolidated into `get_probes`.

Reply:
Do _not_ do the above if we proceed with the fallback.

Reply:
I don't think we need a fallback. If there's no extra pings.yaml it's only the default pings anyway